### PR TITLE
HG-3 and HG-3-SL Plume Fixes

### PR DESCRIPTION
**Pull request by Ash19256**
Making the plumes actually work for more than one engine model, and making the sea-level plume noticeably different from the vacuum plume.

### CODE REVIEW

**Comment by NathanKell** [comment]
I'm confused--there's both vac and SL configs on both the vac and SL parts?

**Comment by Ash19256** [comment]
My response to the SHIP parts was to make it so that both of the SHIP HG-3 parts used the universal config. This had the effect of making it so that you could theoretically configure the "Vacuum" model as if it were a sea level variant and vice versa. As such, plume configs would logically be necessary.

File: GameData/RealismOverhaul/RealPlume_Configs/SXT/SXTHG-3.cfg [hunk 1]
```diff
@@ -0,0 +1,42 @@
+@PART[SXTLT80]:FOR[RealPlume]:NEEDS[SmokeScreen]
```
Comment (PhineasFreak) [inline GameData/RealismOverhaul/RealPlume_Configs/SXT/SXTHG-3.cfg]: Wrong part config name, should be "RO-HG-3".

**Comment by Ash19256** [comment]
Well, that fixes that small issue, but unfortunately the SHIP plumes don't work for some reason. When in the game, the engines appear to default to what appears to be the SXT plume for some reason. I'm not sure if this is something that would have been fixed by correcting the part that the SXT plume directs to, but I'm not sure.

**Comment by Ash19256** [comment]
Alright, as far as I can tell everything is working correctly. I'll do some testing and if nothing's broken (ie. engines using wrong plumes) I think this is ready to be merged.

**Comment by PhineasFreak** [comment]
This is good for a merge.

**Pull request merged**